Makefile.am

```diff
@@ -235,6 +235,55 @@ dist_bin_SCRIPTS += eos-updater-prepare-volume/eos-updater-prepare-volume
 
 dist_man8_MANS += eos-updater-prepare-volume/docs/eos-updater-prepare-volume.8
 
+# Test helper library
+noinst_LTLIBRARIES += test-common/libeos-updater-test-common.la
+
+test_common_libeos_updater_test_common_la_CPPFLAGS = \
+	$(AM_CPPFLAGS) \
+	-DOSTREE_BINARY=\""$(OSTREE)"\" \
+	-DGPG_BINARY=\""$(GPG)"\" \
+	-DOSTREE_WITH_AUTOCLEANUPS \
+	-DG_LOG_DOMAIN=\""eos-updater-test-common"\" \
+	-DLOCALSTATEDIR=\""$(localstatedir)"\"
+	-I$(top_srcdir) \
+	-I$(top_builddir) \
+	-include "config.h" \
+	-I$(top_srcdir)/src \
+	-I$(top_builddir)/src \
+	$(NULL)
+test_common_libeos_updater_test_common_la_CFLAGS = \
+	$(STD_CFLAGS) \
+	$(CODE_COVERAGE_CFLAGS) \
+	$(WARN_CFLAGS) \
+	$(FLATPAK_CFLAGS) \
+	$(GIO_CFLAGS) \
+	$(OSTREE_CFLAGS) \
+	$(NULL)
+test_common_libeos_updater_test_common_la_LDFLAGS = \
+	$(WARN_LDFLAGS) \
+	$(NULL)
+test_common_libeos_updater_test_common_la_LIBADD = \
+	$(CODE_COVERAGE_LIBS) \
+	$(FLATPAK_LIBS) \
+	$(GIO_LIBS) \
+	$(OSTREE_LIBS) \
+	$(top_builddir)/libeos-updater-util/libeos-updater-util-@EUU_API_VERSION@.la \
+	$(NULL)
+test_common_libeos_updater_test_common_la_SOURCES = \
+	test-common/flatpak-spawn.c \
+	test-common/flatpak-spawn.h \
+	test-common/spawn-utils.c \
+	test-common/spawn-utils.h \
+	test-common/misc-utils.c \
+	test-common/misc-utils.h \
+	test-common/ostree-spawn.c \
+	test-common/ostree-spawn.h \
+	test-common/eos-test-utils.c \
+	test-common/eos-test-utils.h \
+	test-common/eos-test-convenience.c \
```

Comment:
(You weren’t to know, but) I have been trying to remove the `eos-` prefix from files as they get moved to subdirectories, since it’s redundant and makes for extra typing and harder tab completion. I’d rename these to `utils.[ch]` and `convenience.[ch]`.